examples/Makefile

```diff
@@ -7,14 +7,6 @@ RELEASE_PATH ?= ../strimzi-$(RELEASE_VERSION)/$(PROJECT_NAME)
 
 release:
 	mkdir -p $(RELEASE_PATH)
-	$(CP) -r ./templates $(RELEASE_PATH)/
-	$(CP) -r ./kafka $(RELEASE_PATH)/
-	$(CP) -r ./kafka-connect $(RELEASE_PATH)/
-	$(CP) -r ./kafka-mirror-maker $(RELEASE_PATH)/
-	$(CP) -r ./kafka-mirror-maker-2 $(RELEASE_PATH)/
-	$(CP) -r ./kafka-bridge $(RELEASE_PATH)/
-	$(CP) -r ./user $(RELEASE_PATH)/
-	$(CP) -r ./topic $(RELEASE_PATH)/
-	$(CP) -r ./metrics $(RELEASE_PATH)/
+	$(CP) -r $$(ls | $(GREP) -Ev '^Makefile$$') $(RELEASE_PATH)/
```

Comment:
the previous version was understandable :-) I have doubts about this one ... but if Tom is happy ;-)

Reply:
If our collective knowledge of `make` is so poor that we're not prepared to do sensible things to make our own lives easier and avoid making stupid errors (i.e. whitelist vs blacklist) then we need to think about whether `make` is really the right tool for the job.

Reply:
Let¨s hope it doesn't need changing for another few years :-o